app/assets/js/rtMedia.backbone.js

```diff
@@ -1787,6 +1862,29 @@ jQuery( document ).ready( function( $ ) {
 function rtmedia_selected_file_list( plupload, file, uploader, error, comment_media_id ) {
 	var icon = '', err_msg = '', upload_progress = '', title = '';
 
+	/**
+	 * Issue 243 fixed: blank error display issue resolved
+	 */
+	if( bp_template_pack && 'legacy' !== bp_template_pack ) {
+
+		if ( 0 < jQuery( '#aw-whats-new-submit' ).length ) {
+
+			jQuery( '#aw-whats-new-submit' ).replaceWith( '<input type="button" id="aw-whats-new-submit" class="button" name="aw-whats-new-submit" value="Post Update">' );
+
+			jQuery( '#aw-whats-new-submit' ).on( 'click', function ( e ) {
+
+				var allowActivityPost = rtMediaHook.call( 'rtmedia_js_before_activity_added', true );
+
+				if( allowActivityPost ) {
```

Comment:
Put `rtMediaHook.call( 'rtmedia_js_before_activity_added', true )` directly inside `if` condition to prevent redundant variable usage.